docs/service-bundle.md

```diff
@@ -0,0 +1,225 @@
+contains a base64 encoded string. This label holds [important
+metadata](https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/developers.md#apb-spec-file)
+related to the service bundle. Example:
+
+```
+version: 1.0        # The version of the Service Bundle Spec being conformed to
+name: example-apb   # The name of the Service Bundle
+description:        # A short description of the Service Bundle
+bindable: True      # Whether this Service Bundle can be bound to other services
+async: optional     # Does this Service Bundle support asynchronous provision
+metadata: 
+  documentationUrl: <link to documentation>
+  imageUrl: <link to URL of image>
+  dependencies:
+  - '<registry>/<organization>/<dependency-name-1>'   
+  - '<registry>/<organization>/<dependency-name-2>'
+  displayName: Example App (APB)
+  longDescription: A longer description of what this APB does
+  providerDisplayName: "Red Hat, Inc."
+plans: # An array of plans supported by this Service Bundle (must have at least 1)
+  - name: default
+    description: A short description of what this plan does
+    free: true
+    metadata:
+      displayName: Default
+      longDescription: A longer description of what this plan deploys
+      cost: $0.00
+    Parameters: # Parameters for this plan
+      - name: parameter_one
+        required: true
+        default: foo_string
+        type: string
+        title: Parameter One
+        maxlength: 63
+      - name: parameter_two
+        required: true
+        default: true
+        title: Parameter Two
+        type: boolean
+```
+
+## Input
+
+The container is run with three arguments.
+
+* Action: a single word such as “provision”. The full list is below.
+* --extra-vars: This is ansible-specific and always has exactly the value
+  “--extra-vars”. An Ansible Playbook Bundle uses this and the third argument
+  as-is when calling ansible by appending them to the ansible command.
+* JSON: a json document of information that is useful to the bundle. Currently
+  it is expressed in a form that is useful for populating variables in ansible
+  that can be used by roles.
+
+### Actions
+
+Possible actions include
+* provision
+* deprovision
+* bind
+* unbind
+* update
+* test
+
+### JSON
+Example of JSON document passed to __provision__:
+
+```
+{
+    "cluster": "openshift",
+    "_apb_plan_id": "default",
+    "_apb_service_class_id": "c23ec213bb8dea1577230c5ce005b9c2",
+    "_apb_service_instance_id": "54636ad3-0378-49a1-a494-f97d0a0daf8e",
+    "mediawiki_admin_pass": "pass",
+    "mediawiki_admin_user": "admin",
+    "mediawiki_db_schema": "mediawiki",
+    "mediawiki_site_lang": "en",
+    "mediawiki_site_name": "MediaWiki",
+    "namespace": "bazproject"
+}
+```
+
+* cluster: the type of cluster on which the service bundle is being run
+  (currently either ‘openshift’ or ‘kubernetes’).
+* _apb_plan_id: the name of a “plan”, as defined through the OSB API, that is
```

Comment:
`_apb` prefixed params need to be reconsidered in a world of generic bundles. The prefix should probably be changed to be agnostic.